### DISCUSSION TITLE
Anyone have luck getting the draft page to populate with your league?

### DISCUSSION THREAD

**Discussion by ASuperGyro** [nmelhado/league-page#310] [Help!]
Not doing anything special as far as I know, just trying to populate the draft page with our league drafts from Sleeper. League as always been in Sleeper so don't need to import across different platforms.

Whatever the default set up is from the master returns a 404 and I can't figure out where I need to adjust things to make it pull from my league. Everything else seems to be set up fine though, I've got my bylaws all in and the manager pages are all set. Every other pull from Sleeper seems in order, I just cant figure the draft boards.

**Comment by nmelhado** [comment]
Can you send over your league ID? I can take a look when I next have a chance. Full disclosure, I have no idea when that will be, but I'll try to get to it in the near future

**Reply by ASuperGyro** [reply]
Yeah I got you, the ID is 1050258497713135616

I'm still going to take time to build out the blog with some content we've been running over the last couple years, so I've got enough to keep me distracted, appreciate the guide you put together as it got me this far with my "coding" knowledge being limited CSS.

Cheers
